### DISCUSSION TITLE
feat request: 添加arm64构建以支持信创机型

### DISCUSSION THREAD

**Discussion by shenmo7192** [SPlayer-Dev/SPlayer#293] [想法 Ideas]
在信创环境下想要听网易云可太难了，之前网易云2.x的时候可以用box64+wine解决，现在3.x加上electron之后wine占用翻倍不说，还经常崩溃，太难了

**Comment by MoYingJi** [comment]
arm64（aarch64）linux 有了喵，但不知道构建出来的包支不支持你的环境安装
